webrtc.html

```diff
@@ -9453,7 +9453,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   </ol>
                 </li>
               </ol>
-              <p>Calling <code><a data-for=
+              <p>Since <code>insertDTMF</code> replaces the tone
+              buffer, in order to add to the DTMF tones to be played,
+              it is necessary to append additional tones to the
+              tone buffer. Calling <code><a data-for=
```

Comment:
Suggestion:  "in order to add to the DTMF tones being played, it is necessary to call <code>insertDTMF</code> with a string containing both the remaining tones (stored in <code>toneBuffer</code>) and the new tones appended together."